src/script/standard.h

```diff
@@ -9,7 +9,7 @@
 #include <script/interpreter.h>
 #include <uint256.h>
 
-#include <boost/variant.hpp>
+#include <variant>
```

Comment:
nit: could probably be sorted below string

Reply:
Thanks, sorted includes